packages/TorneloScoresheet/src/pages/ResultDisplay/ResultDisplay.tsx

```diff
@@ -22,15 +22,24 @@ const ResultDisplay: React.FC = () => {
   }`;
 
   const handleEmailGame = (): void => {
-    console.log('email game button pressed');
     const subject =
       'Tornello Game Results ' +
       infoString +
       ':' +
       resultDisplayMode?.pairing.players[0].firstName +
       ' vs' +
       resultDisplayMode?.pairing.players[1].firstName;
-    const body = '' + resultDisplayMode?.pairing.pgn;
+    const body =
+      'Tornello Game Results ' +
+      infoString +
+      '\n\nPlayer 1: ' +
+      resultDisplayMode?.pairing.players[0].firstName +
+      resultDisplayMode?.pairing.players[0].lastName +
+      '\nPlayer 2: ' +
+      resultDisplayMode?.pairing.players[1].firstName +
+      resultDisplayMode?.pairing.players[1].lastName +
+      '\n\nFull Game PGN:\n' +
+      resultDisplayMode?.result.gamePgn;
     sendEmail('', subject, body); //option to add arbiter 'To' address in the future
```

Comment:
We aren't setting the email here?
We need to have a popup sheet come up when we press 'email game' that asks the user to input an email address.
Once the user clicks send on that popup, then we send the email with that email address

Reply:
Sure, I've added this now. I must have misunderstood the requirement since in the Trello it said "The players are obliged to submit the text of their game to the Organizer with reference to article 8.3 of the Laws of Chess. For the MVP, it is sufficient to have a button that emails the file to Tornelo" so I assumed it would just send to one email address. I've just made it so that it asks the user to input their desired email address :)